langfuse-core/src/index.ts

```diff
@@ -726,5 +755,19 @@ export class LangfuseEventClient extends LangfuseObservationClient {
   }
 }
 
+export class LangfusePromptClient {
+  private promptResponse: CreateLangfusePromptResponse;
+  constructor(prompt: CreateLangfusePromptResponse) {
+    this.promptResponse = prompt;
+  }
+
+  compile(variables: unknown): string {
+    if (typeof variables !== "object") {
+      throw new Error("Variables must be an object");
+    }
+    return mustache.render(this.promptResponse.prompt, variables);
```

Comment:
This will replace undefined variables with empty strings.
Maybe one should add different modes with erroring out on missing variables or doing a partial render like 
https://js.langchain.com/docs/modules/model_io/prompts/partial